tests/unit/test_charm.py

```diff
@@ -284,8 +284,8 @@ def test__is_valid_config(
     act: when _is_valid_config is called
     assert: then the expected configuration validity and message is returned.
     """
-    harness.charm._stored.relation_configured = relation_configured
-    harness.charm._stored.relation_agent_tokens = agent_tokens
+    harness.charm._stored.relation_configured = relation_configured  # type: ignore
+    harness.charm._stored.relation_agent_tokens = agent_tokens  # type: ignore
```

Comment:
Would the type ignore also be due to mypy reporting function overloading?

Reply:
yes